src/DssSpellCollateral.sol

```diff
@@ -30,99 +30,87 @@ contract DssSpellCollateralAction {
     // A table of rates can be found at
     // https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
-
+    uint256 internal constant ONE_FIVE_PCT_RATE = 1000000000472114805215157978;
+    
     // --- Math ---
-    // uint256 constant THOUSAND   = 10 ** 3;
+    uint256 constant THOUSAND   = 10 ** 3;
     uint256 constant MILLION    = 10 ** 6;
     // uint256 constant BILLION    = 10 ** 9;
 
-    bytes32 constant RWA007_A                 = "RWA007-A";
-    uint256 constant RWA007_A_AUTOLINE_AMOUNT = 250 * MILLION;
-    uint256 constant RWA007_A_AUTOLINE_GAP    =  50 * MILLION;
-    uint256 constant RWA007_A_AUTOLINE_TTL    = 1 weeks;
-
     // --- DEPLOYED COLLATERAL ADDRESSES ---
-    // address constant XXX                  = 0x0000000000000000000000000000000000000000;
-    // address constant PIP_XXX              = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_JOIN_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_CALC_XXX_A  = 0x0000000000000000000000000000000000000000;
-
-    // --- Offboarding: Current Liquidation Ratio ---
-    // uint256 constant CURRENT_XXX_A_MAT              =  XYZ * RAY / 100;
-
-    // --- Offboarding: Target Liquidation Ratio ---
-    // uint256 constant TARGET_XXX_A_MAT               =  XYZ * RAY / 100;
-
+    address internal constant RETH                 = 0xae78736Cd615f374D3085123A210448E74Fc6393;
+    address internal constant PIP_RETH             = 0xeE7F0b350aA119b3d05DC733a4621a81972f7D47;
```

Comment:
This deployer (`0x39abd7819e5632fa06d2ecbba45dca5c90687ee3`) is still relied.  

1. This address needs to be added to the `addresses_deployers.sol`
2. We need to have a carve out of the `checkWards` that allows this one deployer to still be authed on this contract as long as the DC is 0